apps/web/src/client/components/file-uploader/_ui.js

```diff
@@ -0,0 +1,109 @@
+/**
```

Comment:
the structure of the client side javascript is very temporary. will very likely change once I've got all the compnoents (ui, validation, errors handling, internal api, blob storage api)